packages/fyndiq-component-alert/src/index.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styles from '../styles.less'
+import { loadstate, saveState } from './localStorage'
```

Comment:
just for the sake of having the same naming convention, you could rename `loadstate` to `loadState` (capital **S**)